js/controllers/dialog.js

```diff
@@ -61,7 +61,13 @@ DialogController.prototype.addButton = function(id, text) {
 };
 
 DialogController.prototype.setText = function(text) {
```

Comment:
It would be good to add some documentation that this is a variable number of arguments. Something like
```
/**
 * @param {...string} text
 */
DialogController.prototype.setText = function(text) {
```

If possible, it at least lets us know that setText is called with multiple arguments.

Reply:
Yeah I was gonna add the JSDoc stuff later once I've got the translated messages.